peewee_async/pool.py

```diff
@@ -8,7 +8,8 @@
 class PoolBackend(metaclass=abc.ABCMeta):
     """Asynchronous database connection pool.
     """
-    def __init__(self, *, database: str, **kwargs: Any) -> None:
+
+    def __init__(self, *, database: Optional[str] = None, **kwargs: Any) -> None:
```

Comment:
Actually we don't pass database as None here and we should not allow optional for database param